js_modules/dagit/packages/core/src/ui/Filters/FilterDropdown.tsx

```diff
@@ -261,6 +348,27 @@ const TextInputWrapper = styled.div`
   }
 `;
 
+type MenuItemProps = React.ComponentProps<typeof _MenuItem> & {
+  menuKey: string;
+  index: number;
+};
+const MenuItem = React.memo(({menuKey, index, ...rest}: MenuItemProps) => {
```

Comment:
Rename to something that doesn't conflict with the `ui` component? I could see this being pretty confusing in the Components dev tools, or for users who are new to this code.